endpointmanager/pkg/archivefile/archivefile_integration_test.go

```diff
@@ -56,6 +56,7 @@ var testMetadata = endpointmanager.FHIREndpointMetadata{
 	Errors:            "Smart Response Failed",
```

Comment:
This reminded me, should we include the requested fhir version in the archive file? or i guess you could just pull the rows in the table where requested_fhir_version is "None" and have the archive file only be those rows.